### DISCUSSION TITLE
Move community governance documents to keptn/community

### DISCUSSION THREAD

**Discussion by oleg-nenashev** [keptn/keptn#9090] [Ideas]
Hi all,

At the moment we have a number of communitymembership/governance repositories in https://github.com/keptn/keptn . It was OK for a monorepo project, but gets more and more complicated with more root projects being added (website, integration services, sub-projects like Lifecycle Controller, etc.) 

## Proposal

* Move the community membership and governance files to https://github.com/keptn/community
* Add tombstone links to files where we want to present links in Keptn

## Scope

* ADOPTERS
* CODE_OF_CONDUCT => Already moved
* CONTRIBUTORS
* GOVERNANCE
* MAINTAINERS

![image](https://user-images.githubusercontent.com/3000480/198522177-9d946677-f24e-4a74-bf04-817196d15001.png)

**Comment by AloisReitbauer** [comment]
Good idea moving. However, as we have no agreed upon governance model I wonder whether we should rather start with a new governance PR rather than moving.

**Reply by oleg-nenashev** [reply]
I am moving the governing too in https://github.com/keptn/keptn/discussions/8991
